tests/dune

```diff
@@ -1,4 +1,19 @@
+(executable
+    (name pp)
+    (modules pp)
+    (libraries ppx_mysql ppxlib))
+
+(rule
+    (targets test_ppx.result)
+    (deps test_ppx.ml)
+    (action (run ./pp.exe --impl %{deps} -o %{targets})))
+
+(alias
+    (name runtest)
+    (action (diff test_ppx.expected.ml test_ppx.result)))
+
 (test
     (name test_query)
+    (modules test_query)
```

Comment:
Could you split it out into sublibraries to avoid modules? I find that a very annoying feature, because for every element you want to add you suddenly require a new boilerplat-ey entry in the corresponding `dune` file.

Reply:
I'll create a separate subdirectory for each test.